src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -13,6 +13,31 @@
                     #{studyId}
                 </foreach>
             </if>
+            <!-- sample-lists-counts filter -->
+            <if test="studyViewFilter.caseLists != null and !studyViewFilter.caseLists.isEmpty()">
+                INTERSECT
+                <!-- get samples that are in each sample list in the filter -->
+                SELECT * FROM (
+                <foreach item="ANDGroup" collection="studyViewFilter.caseLists" separator="INTERSECT">
+                    SELECT sv.sample_unique_id
+                    FROM sample_list_list sll
+                        LEFT JOIN sample_view sv ON sll.sample_id=sv.internal_id
+                        LEFT JOIN sample_list sl on sll.list_id=sl.list_id
+                    <where>
```

Comment:
@haynescd @onursumer am i right that we don't need to check for study in each filter?  this should only be done once, right?   @gblaih i may have mislead you with my own filter that i think does this also.